{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -147,6 +150,38 @@ search = "\"version\": \"{current_version}\""
 replace = "\"version\": \"{new_version}\""
 {%- endif %}
 
+[[tool.bumpversion.files]]
+filename = "CHANGES.rst"
+search = """\
+`Unreleased <https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}>`_ (latest)
+================================
+{%- for _ in range((cookiecutter.github_username | length) + (cookiecutter.project_slug | length)) %}={% endfor -%}
+=============
+
+"""
+replace = """\
+`Unreleased <https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}>`_ (latest)
+================================
+{%- for _ in range((cookiecutter.github_username | length) + (cookiecutter.project_slug | length)) %}={% endfor -%}
+=============
+
+Changes:
+--------
+- No change.
+
+Fixes:
+------
+- No change.
+
+.. _changes_{new_version}:
+
+`{new_version} <https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}/tree/{new_version}>`_
+===========================
+{%- for _ in range((cookiecutter.github_username | length) + (cookiecutter.project_slug | length)) %}={% endfor -%}
+=======
+
+"""
+
```

Comment:
You could also add the `Changes` and `Fixes` entries to the initial template so that we don't get empty space between `0.1.0` and the entry above it.

Reply:
@bzah

I've resolved everything except my first comment here. It might not be supported, in which case this could be complicated.

`bump-my-version` has a very active maintainer, so it might be worth opening an issue?